plugins/inputs/openweathermap/openweathermap_test.go

```diff
@@ -293,38 +294,46 @@ func TestForecastGeneratesMetrics(t *testing.T) {
 		testutil.MustMetric(
 			"weather",
 			map[string]string{
-				"city_id":  "2988507",
-				"forecast": "3h",
-				"city":     "Paris",
-				"country":  "FR",
+				"city_id":        "2988507",
```

Comment:
No change required. I'm sorta surprised we let `city_id` be a tag and not a field. Must have added `city` later.